tf/requirements-gpu.txt

```diff
@@ -1,8 +1,8 @@
 antlr4-python3-runtime==4.7
 jupyter==1.0.0
-numpy==1.14.5
+numpy==1.16.4
```

Comment:
Do you check the code already in the repo runs with numpy 1.16?

Reply:
I was unable to run the example ProtoNN/Bonsai code without this change. Besides there's some issue in the existing requirements files for cpu and gpu. 

See 
https://github.com/microsoft/EdgeML/blob/master/tf/requirements-gpu.txt
https://github.com/microsoft/EdgeML/blob/master/tf/requirements-cpu.txt

The cpu version has numpy==1.16.4 while gpu has numpy==1.14.5

Also, there's a notion of least numpy variant which tensorflow 1.15 supports. As per the dependencies mentioned in the tensorflow setup, numpy version should be above 1.16 ( https://github.com/tensorflow/tensorflow/blob/r1.15/tensorflow/tools/pip_package/setup.py )